akka/shopping-cart-java/src/main/java/shopping/cart/Main.java

```diff
@@ -26,7 +26,7 @@ public class Main {
 
   // tag::SendOrderProjection[]
   public static void main(String[] args) {
-    ActorSystem<Void> system = ActorSystem.create(Behaviors.empty(), "ShoppingCartService");
+    ActorSystem<Void> system = ActorSystem.create(Behaviors.empty(), "shopping-cart-service");
```

Comment:
Was this a requirement, or more a convention?

Reply:
Check the Akka Guide for changes. From https://github.com/akka/akka-guide/pull/832, so could be to simplify bootstrap in Kubernetes? (by having the service and actor system names the same)